04-Enumerable/Enumerable/Task.cs

```diff
@@ -457,9 +512,10 @@ public class Task {
         ///   { null } => 0
         ///   { } => 0
         /// </example>
-        public int GetTotalStringsLength(IEnumerable<string> data) {
+        public int GetTotalStringsLength(IEnumerable<string> data)
+        {
             // TODO : Implement GetTotalStringsLength
-            throw new NotImplementedException();
+            return data.Sum(x => string.IsNullOrEmpty(x)?0:x.Length);
```

Comment:
if you use symbols for if-else statement, please use spaces for these characters, or split into lines, without spaces it looks like one long name of smth